.travis.yml

```diff
@@ -2,11 +2,10 @@ language: scala
 scala:
 # When updating Scala versions, also check the excludes
 # in build matrix below.
-- 2.11.12
```

Comment:
Maintain `2.11.12`. We are still supporting Scala 2.11.

Reply:
I would say cross-build. The way I see the templates are serving two purposes:

1. As sample applications for users.
1. As a last (late) validation for Play releases.

Considering the second point, it is good that we cross-build.